sources/osg/Material.js

```diff
@@ -96,6 +96,15 @@ define( [
             return this._shininess;
         },
 
+        setShadeless: function ( a ) {
```

Comment:
Not sure I want this, material is supposed to be like material of opengl. could you explain the link / logic behind this. I guess it's because of shader vertex to remove some attributes, but shadeless could be detected with no light in state during the shagerGenerator invocation ?